integration-tests/features/steps/stack_analysis.py

```diff
@@ -199,49 +199,32 @@ def check_stack_analyses_response(context, url):
     jsonschema.validate(resp_json, schema)
 
 
-@when('I download and parse outlier probability threshold value')
-def download_and_parse_outlier_probability_threshold_value(context):
-    """Download and parse outlier probability threshold value.
+def check_frequency_count(usage_outliers, package_name):
+    """Check the frequency count attribute.
 
-    This Special step that is needed to get the stack analysis outlier
-    probability threshold.
+    Try to find frequency count value for given package and check that
+    the value is within permitted range.
     """
-    content = download_file_from_url(STACK_ANALYSIS_CONSTANT_FILE_URL)
-    context.outlier_probability_threshold = parse_float_value_from_text_stream(
-        content, STACK_ANALYSIS_OUTLIER_PROBABILITY_CONSTANT_NAME)
+    # TODO: uncomment this after the https://github.com/openshiftio/openshift.io/issues/843
+    # will be merged
+    if True:
+        pass
 
-
-@then('I should have outlier probability threshold value between {min:f} and {max:f}')
-def check_outlier_probability_threshold_value(context, min, max):
-    """Check that the outlier probability falls between selected range."""
-    v = context.outlier_probability_threshold
-    assert v is not None
-    assert v >= min
-    assert v <= max
-
-
-def check_outlier_probability(usage_outliers, package_name, threshold_value):
-    """Check the outlier probability.
-
-    Try to find outlier probability for given package is found and that
-    its probability is within permitted range.
-    """
-    # NOTE: there's a typo in the attribute name (issue #73)
-    # the following line should be updated after the issue ^^^ will be fixed
-    outlier_probability_attribute = "outlier_prbability"
+    frequency_count_attribute = "frequency_count"
 
     for usage_outlier in usage_outliers:
         if usage_outlier["package_name"] == package_name:
-            assert outlier_probability_attribute in usage_outlier, \
+            assert frequency_count_attribute in usage_outlier, \
                 "'%s' attribute is expected in the node, " \
-                "found: %s attributes " % (outlier_probability_attribute,
+                "found: %s attributes " % (frequency_count_attribute,
                                            ", ".join(usage_outlier.keys()))
-            probability = usage_outlier[outlier_probability_attribute]
-            assert probability is not None
-            v = float(probability)
-            assert v >= threshold_value and v <= 1.0, \
-                "outlier_prbability value should fall within %f..1.0 range, "\
-                "found %f value instead" % (threshold_value, v)
+            value = usage_outlier[frequency_count_attribute]
+            assert value is not None
+            v = int(value)
```

Comment:
Can be something like:
```python
try:
     v = int(value)
except ValueError as v :
       raise Exception("Invalid value found for frequency_count for the package {p}".format(p=package_name))

Reply:
yup, but I wanted to distinguish between missing attribute, null value and improper int value (negtive integer in this simple case)

Reply:
yes, but it won't be explicitly displayed in the message - people will have to look at stack trace